.travis.yml

```diff
@@ -107,6 +107,38 @@ matrix:
       name: "Test blinky example - debug (NRF52840_DK)"
       env: EXAMPLE_NAME=mbed-os-example-blinky TARGET_NAME=NRF52840_DK PROFILE=debug CACHE_NAME=debug-NRF52840_DK
 
+    - <<: *mbed-tools-test
```

Comment:
Do we need to test `K64F - develop` or do we already test that one somewhere?

Reply:
Added test for `develop` profile with `K64F`.